tests/utils/runtimeclass/runtimeclass.go

```diff
@@ -0,0 +1,57 @@
+package runtimeclass
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	v1 "k8s.io/api/node/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	retryInterval = 5
+)
+
+func DefineRunTimeClass(name string) *v1.RuntimeClass {
+	return &v1.RuntimeClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Handler: "runc",
+	}
+}
+
+func CreateRunTimeClass(rtc *v1.RuntimeClass) error {
```

Comment:
Based on the location of similar functions in the project I think both CreateRunTimeClass and isRtcCreated should likely be moved into a runtimeclass.go file in the globalhelper package (for other k8s objects, utils contains definitions and globalhelper has creation functions)

Reply:
Agree, thanks, done.